### DISCUSSION TITLE
收集：开源项目中经常使用的英文缩写词

### DISCUSSION THREAD

**Discussion by LinuxSuRen** [LinuxSuRen/open-source-best-practice#136] [Open Source Cases]
对英文不是很熟悉或初次接触开源贡献的人来说，可能会对一些英文缩写词不是很了解具体的含义，下面是部分缩写词以及解释（欢迎补充）：

* BTW - by the way
* cc
* LGTM - look good to me
* BDFL - 终身仁慈独裁者（ Benevolent Dictator For Life）
* TSC - Technical Steering Committee

**Comment by wey-gu** [comment]
PTAL = please take a look

**Comment by pacoxu** [comment]
- ACK
 - EOL: end of life 不服务了😄
 - ASAP
 - IMO 
 - FYI 请知悉 for your information
 - IIRC 如果我记得没错的话，If I remember correctly
 - PR
 - CI: Continuous integration 
 

K8/CNCF 特有的几个
 - TAG:  Technical Advisory Group. 目前CNCF 工作组
 - SIG: Special Interest Group. 目前 Kubernetes 分组方式
 - WG： work group 这个一半是交叉SIG/TAG 的时候用
 - KEP: Kubernetes Enhancement Proposals。就是设计稿。
 

暂时只想到这些🤣
